samples/XAMLCatalog/XAMLCatalog.vsimporter/XAMLCatalog-WinStore10/XAMLCatalog-Debug-xcvars.txt

```diff
@@ -85,15 +85,15 @@ PRODUCT_BUNDLE_IDENTIFIER = Microsoft.XAMLCatalog
 PRODUCT_NAME = XAMLCatalog
 PRODUCT_TYPE = com.apple.product-type.application
 PROJECT = XAMLCatalog
-PROJECT_DIR = I:/enl/vso_1_WinObjC/samples/XAMLCatalog
-PROJECT_FILE_PATH = I:\enl\vso_1_WinObjC\samples\XAMLCatalog\XAMLCatalog.xcodeproj
+PROJECT_DIR = E:/GIT_REPOS/winobjc/samples/XAMLCatalog
+PROJECT_FILE_PATH = E:\GIT_REPOS\winobjc\samples\XAMLCatalog\XAMLCatalog.xcodeproj
```

Comment:
Yes, every time you use VSIMPORTER, these changes are automatically made on your behalf. We could revert them, as Yi suggested, but it's going to resurface each time someone else makes a change.

---
In reply to: [97601767](https://github.com/Microsoft/WinObjC/pull/1771#discussion_r97601767) [](ancestors = 97601767)